Dockerfile

```
@@ -4,7 +4,18 @@ FROM composer/composer:2-bin AS composer
 FROM mlocati/php-extension-installer:2.2 AS php_extension_installer
 
 # Build Caddy with the Mercure and Vulcain modules
-FROM caddy:2.9-builder-alpine AS app_caddy_builder
+FROM caddy:2.10-builder-alpine AS app_caddy_builder
+#!/usr/bin/env bash
+# Verify that the pinned digest matches caddy:2.10-builder-alpine
+ENV EXPECTED=3c46452d42e725e6c62ac11cbc91d1d6e4c53612eafe97f948aaa63b53ee9b02
+
+RUN ACTUAL=$(curl -sL https://raw.githubusercontent.com/caddyserver/caddy-docker/aa3e73e0731fbca665be75edef9fbb60d3169278/2.10/builder/Dockerfile | sha256sum | awk '{ print $1 }') && \
```

Hadolint (2.12.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)